docs/feature_combo.md

```diff
@@ -65,26 +65,26 @@ Additionally, this example shows how you can leave `COMBO_COUNT` undefined.
 
 ```c
 enum combo_events {
-  TH_THE,
+  EM_EMAIL,
   BSPC_LSFT_CLEAR,
   COMBO_LENGTH
 };
 uint16_t COMBO_LEN = COMBO_LENGTH; // remove the COMBO_COUNT define and use this instead!
 
-const uint16_t PROGMEM the_combo[] = {KC_BSPC, KC_T, COMBO_END};
+const uint16_t PROGMEM email_combo[] = {KC_BSPC, KC_T, COMBO_END};
```

Comment:
```suggestion
const uint16_t PROGMEM email_combo[] = {KC_E, KC_M, COMBO_END};
```

Reply:
Such small change and I still miss something... :sweat_smile: